reference-implementation/lib/writable-stream.js

```diff
@@ -173,11 +176,26 @@ function WritableStreamFinishClose(stream) {
   defaultWriterClosedPromiseResolve(stream._writer);
 }
 
-function WritableStreamFulfillWriteRequest(stream) {
-  assert(stream._writeRequests.length > 0);
+function WritableStreamRejectUnresolvedPromises(stream) {
```

Comment:
WritableStreamRejectPromisesInReactionToError? Not important, but maybe an improvement.